TruckMuncher/MapViewController.swift

```diff
@@ -337,6 +350,9 @@ class MapViewController: UIViewController,
                 self.navigationItem.titleView?.alpha = self.showingMenu ? 0.0 : 1.0
                 self.navigationItem.leftBarButtonItem?.tintColor = color.colorWithAlphaComponent(self.showingMenu ? 0.0 : 1.0)
                 self.navigationItem.rightBarButtonItem?.tintColor = color.colorWithAlphaComponent(self.showingMenu ? 0.0 : 1.0)
+            }, completion: { (completed) -> Void in
+                currentView.updateViewWithColor(self.showingMenu ? primaryColor : carouselBackground)
```

Comment:
Yeah, thats why it was down in the completion block. I tried messing around with the `reloadSections(: withRowAnimation:)` method this morning and it doesn't look very good either.